### DISCUSSION TITLE
List of topics for publisher declarations

### DISCUSSION THREAD

**Discussion by dimbler** [ag2ai/faststream#1215] [Ideas]
Dear Team,
Declaring publisher it's a great point for simple creating application/testing, but sometimes have a requirements create multiple publishers for different topics
```
for topic in settings['kafka_source_topics']:
    router.subscriber(topic)(handle)
```
Could you implement list of topics instead of creating iterator?

**Comment by davorrunje** [comment]
Hi! This is already implemented, please check out the Reference:

https://faststream.airt.ai/latest/api/faststream/confluent/KafkaBroker/#faststream.confluent.KafkaBroker.subscriber

so you could write something like this:
```python
router.subscriber(*settings['kafka_source_topics'])(handle)
```
